distribution/docs/src/main/resources/content/_reference/_tables/AttributeSuggestionsWhitelist.adoc

```diff
@@ -0,0 +1,27 @@
+:title: Catalog UI Search Attribute Suggestion Whitelist
+:id: org.codice.ddf.catalog.ui.security.facetwhitelist
+:type: table
+:status: published
+:application: ${ddf-ui}
+:summary: Catalog UI Search Attribute Suggestion Whitelist
+
+.[[_org.codice.ddf.catalog.ui.security.facetwhitelist]]Catalog UI Search Attribute Suggestion Whitelist
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Attribute Suggestion Whitelist
+|facetAttributeWhitelist
+|String
+|List of attributes that will query the backend and provide a list of suggestions that gets populated in the dropdown in a query form.
+Caution: These attributes will not be run through security.
```

Comment:
✏️ 	Is there a way to make this caution line stand out a little more?

Reply:
I could bold/italicize it, or change it to another text color (red or orange maybe). I'm hesitant to do so since other documentation leaves similar messages as plaintext.

Reply:
If others are doing the same, then this is fine

Reply:
1) no default value?
2) Consider this rewording:
List of attributes that will provide autocomplete suggestions. Suggestions are generated from popular values for the attribute in the data.
Caution: Suggestion values are not protected by any security. Only chose attributes whose popular values will be safe for all users to view.

Reply:
consider adding a screenshot of the autocomplete in action

Reply:
1. Default Value is an empty list. Other configurations with empty lists as default leave that cell empty
2. Updated in the docs and in the metatype file.
Screenshot added from previous PR